Documentation/devicetree/bindings/sound/everest,es8326.yaml

```diff
@@ -0,0 +1,115 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/sound/everest,es8326.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Everest ES8326 audio CODEC
+
+maintainers:
+  - Daniel Drake <drake@endlessm.com>
+  - David Yang <yangxiaohua@everest-semi.com>
+
+properties:
+  compatible:
+    const: everest,es8326
+
+  reg:
+    maxItems: 1
+
+  clocks:
+    items:
+      - description: clock for master clock (MCLK)
+
+  clock-names:
+    items:
+      - const: mclk
+
+  "#sound-dai-cells":
+    const: 0
+
+  mclk-rate:
+    description:
+      clock rate of mclk
+    $ref: /schemas/types.yaml#/definitions/uint32-array
+    minItems: 1
+
+  jack-pol:
+    description:
+      just the value of reg 57. Bit(3) decides whether the jack polarity is inverted.

```

Comment:
how does this work since you also have the 'jack-inverted' value?

Reply:
A default value is defined here which can be inverted by 'jack-inverted' ？